docs/providers/aws/events/cloudfront.md

```diff
@@ -238,6 +238,20 @@ functions:
             id: 658327ea-f89d-4fab-a63d-7e88639e58f6 # references AWS Managed Policy named Managed-CachingOptimized
 ```
 
+It is also possible to reference policies with `behavior.CachePolicyId` property. When both `cachePolicy.(id|name)` and `behavior.CachePolicyId` are specified, setting from `cachePolicy.(id|name)` will be used.
```

Comment:
I think instead of presenting it this way, we could add a sentence like this. 

```
Similarily, when `cachePolicy.name` and `behavior.CachePolicyId` are specified, setting from `cachePolicy.name` will be used. 
```

The reason for being more descriptive here is that I don't think we use similar notation in our documentation anywhere and I think in such cases it's better to be a little bit more verbose. Let me know what do you think :bow:

Reply:
Fully agree with this. Thanks for the guidance 🙇  I will update the PR and let you know.